CHANGELOG.md

```diff
@@ -36,6 +59,18 @@ These should not affect the functionality of the module.
 - Fix Ubuntu [\#35](https://github.com/voxpupuli/puppet-openvmtools/pull/35) ([ghoneycutt](https://github.com/ghoneycutt))
 - modulesync 3.0.0 & puppet-lint updates [\#31](https://github.com/voxpupuli/puppet-openvmtools/pull/31) ([bastelfreak](https://github.com/bastelfreak))
 
+## [epel-dependency](https://github.com/voxpupuli/puppet-openvmtools/tree/epel-dependency) (2020-04-29)
```

Comment:
this is odd. why do we now see branches in the changelog. that shouldn't be the case :thinking: . let me delete them

Reply:
edit: oh that branch does not even exist in our repo

Reply:
It also does not exist in my fork or local checkout...

Reply:
https://github.com/voxpupuli/puppet-openvmtools/tree/epel-dependency DOES exist

Reply:
vey odd, but I deleted it:
```
$ git push origin :epel-dependency
To github.com:voxpupuli/puppet-openvmtools.git
 - [deleted]         epel-dependency
```

Reply:
Its gone but there is still the branch ubuntu-20-support 
https://github.com/voxpupuli/puppet-openvmtools/pull/54/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR41